primitives/src/authorization.rs

```diff
@@ -86,6 +87,16 @@ pub struct Authorization<U> {
     pub auth_id: u64,
 }
 
+/// Authorization Identity data
+#[derive(Encode, Decode, Default, Clone, PartialEq, Eq, Debug, PartialOrd, Ord)]
+pub struct JoinIdentityData {
+    /// Target DID under which signing_item need to be added
+    pub target_did: IdentityId,
+
+    /// Signing Item
+    pub signing_item: Option<SigningItem>,
```

Comment:
we don't need the full `signing_item` here. We just need an array of permissions.